docs/Catalog/LocalMode.md

```diff
@@ -2,12 +2,15 @@
 <!-- markdownlint-disable -->
 
 You can securely and privately run the Quilt catalog in "single-player mode" on your machine.
+
+<!-- SK: What is the catalog? -->
```

Comment:
should we address these comments in the current PR or merge them as they are? (i have a feeling they will hang in there forever if we do this)

Reply:
Thank you, I'll work to remove we can discuss separately.